reviews/models/mixins.py

```diff
@@ -182,17 +192,83 @@ def resubmission_allowed(self, ev):
         return self.reviewable.provider.reviews_workflow == workflow.Workflows.PRE_MODERATION.value
 
     def notify_submit(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        info = self.get_info()
+        context = self.get_context()
+        context['referrer'] = ev.kwargs.get('user')
+        context['template'] = 'reviews_submission_confirmation'
+        context['is_pre_moderation'] = info.get('is_pre_moderation')
+        reviews.reviews_email.send(context=context, caller='notify_submit')
 
     def notify_accept(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        self.get_accept_reject_notification()
 
     def notify_reject(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        self.get_accept_reject_notification()
 
     def notify_edit_comment(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        context = self.get_context()
+        info = self.get_info()
+        context['template'] = 'reviews_update_comment'
+        if info.get('notify_comment'):
+            reviews.reviews_email.send(context=context, caller='notify_edit_comment')
+
+    def get_info(self):
+        return {
+            'is_rejected': self.action.to_state == workflow.States.REJECTED.value,
+            'notify_comment': not self.reviewable.provider.reviews_comments_private and self.action.comment,
+            'is_pre_moderation': self.reviewable.provider.reviews_workflow == workflow.Workflows.PRE_MODERATION.value
+        }
+
+    def get_accept_reject_notification(self):
+        info = self.get_info()
+        context = self.get_context()
+        context['template'] = 'reviews_submission_status',
+        context['is_pre_moderation'] = info.get('is_pre_moderation')
+        context['notify_comment'] = info.get('notify_comment')
+        context['is_rejected'] = info.get('is_rejected')
+        reviews.reviews_email.send(context=context, caller='accept_reject_notification')
+
+    def get_context(self):
+        return {
+            'creator': self.reviewable.node.creator,
+            'email_recipients': [contributor._id for contributor in self.reviewable.node.contributors],
+            'settings': settings.DOMAIN,
+            'node': self.reviewable.node,
+            'reviewable_title': self.reviewable.node.title,
```

Comment:
Nitpick: Maybe remove the context values that contain no logic, and just pass in `reviewable` (and maybe `node`, if you want) instead. If the template had `reviewable.node.title` or `node.title` instead of `reviewable_title`, it might be easier to read and understand. Same for `creator`, `reviewable_url`, `provider_comments_private`, `provider`, `provider_preprint_word`